product_owners.md

```diff
@@ -0,0 +1,37 @@
+# Product Owners
+
+We strive for each of our applications to have a product owner (PO). The product
+owner is usually a key stakeholder for the application who can collaborate
+closely with us to guide development. Documentation of our team's [work cycles](/work_cycles.md) and [technical liaison role](/technical_liaisons.md) should give product owners useful context for this collaboration.
+
+The exact practices of each product owner will vary according to the application and the
+individuals themselves. Generally, product owners will work to:
+
+* Maintain a list of the highest priority issues for stakeholders on an ongoing
+basis.
+  * We recommend keeping track of these either in Github using a list of 10
+  "On Deck" labeled issues or using Zenhub to organize these to the top of the
```

Comment:
On deck: is it 10 or 15? I have a ghost of a memory of “15” being mentioned at one point.

Reply:
I told Pulfalight 10 (they used to get 6), and I kind of like the idea of "top 10 issues." 15 is probably more than we can get to in a cycle.